docker/Dockerfile

```diff
@@ -4,24 +4,13 @@ LABEL org.opencontainers.image.source="https://github.com/syslog-ng/syslog-ng"
 
 ARG PKG_TYPE=stable
 
-RUN apt-get update -qq && apt-get install -y \
-    wget \
-    ca-certificates \
-    gnupg2 \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | gpg --dearmor > /usr/share/keyrings/ose-repo-archive-keyring.gpg && \
-  echo "deb [signed-by=/usr/share/keyrings/ose-repo-archive-keyring.gpg] https://ose-repo.syslog-ng.com/apt/ ${PKG_TYPE} debian-testing" | tee --append /etc/apt/sources.list.d/syslog-ng-ose.list
-
-RUN apt-get update -qq && apt-get install -y \
-    libdbd-mysql libdbd-pgsql libdbd-sqlite3 syslog-ng libjemalloc2 \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt update -qq && apt install -y wget ca-certificates gnupg2 libdbd-mysql libdbd-pgsql libdbd-sqlite3 syslog-ng libjemalloc2 && rm -rf /var/lib/apt/lists/*&& rm -rf /var/lib/apt/lists/* && \
+wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | gpg --dearmor > /usr/share/keyrings/ose-repo-archive-keyring.gpg && \
+echo "deb [signed-by=/usr/share/keyrings/ose-repo-archive-keyring.gpg] https://ose-repo.syslog-ng.com/apt/ ${PKG_TYPE} debian-testing" | tee --append /etc/apt/sources.list.d/syslog-ng-ose.list
```

Comment:
Please keep those line breaks. Readability is also important.

Reply:
Readability is not changed here; its even improved slightly.One can always set line-wrap to the same effect. So I dont understand your objection(s).

Reply:
Okay. Since this is only a matter of taste, I'm marking my comment as resolved.